packages/ember-metal/lib/properties.js

```diff
@@ -112,7 +112,19 @@ if (EMBER_METAL_ES5_GETTERS) {
         get(descriptor, property) {
           if (property === DESCRIPTOR) {
             return descriptor;
-          } else if (property === 'toString' || property == 'valueOf' || (Symbol && property === Symbol.toPrimitive)) {
+          } else if (
+            property === 'prototype' ||
+            property === 'constructor' ||
+            property === 'nodeType'
```

Comment:
Are these not needed for the other branch (no native `Proxy`) as well?

Reply:
Everything returns undefined by default in that branch, we only add assertions for the specific properties that we know about...